extensions/app-runtime.md

```diff
@@ -0,0 +1,51 @@
+# Application Runtime Extensions
+
+This document specifies a contract between a platform and processes running in an application OCI image.  
+
+## Runtime Environment Variables
+
+The following environment variables extend the [Environment - Provided by the Platform](../buildpack.md#provided-by-the-platform) section in the Buildpack Interface Specification.
+The are exclusively available during the launch phase.
+
+| Env Variable         | Example Value                    | Description                       | Mandatory
+|----------------------|----------------------------------|----------------------------------------------
+| `PORT`               | `8080`                           | port for primary web process type | [x]
+| `CNB_APP_ROUTES`     | `{"web":{"http":{"port":8080}}}` | routes for each process type      | [x]
+| `CNB_APP_INDEX`      | `0`                              | app instance index (logical)      | [x] 
+| `CNB_APP_NAME`       | `example-app`                    | app name                          |
+
+### Format: `CNB_APP_ROUTES`
+
+The `CNB_APP_ROUTES` enviroment variable MUST be JSON-formatted and follow this schema:
+
+```
+{
+    "<process type>": {
```

Comment:
Is the `<process type>` required at this point?  Since this variable is only available at runtime, it is only exposed to a single process type and therefore only a single one of these stanzas is pertinent.

Reply:
That's correct, I'll remove it.

Reply:
I think we also talked about the `CNB_APP_INDEX` being allowed to contain the `<process type>` (on Heroku for example this will be a value like `web.1`), which makes it redundant here.

Reply:
Both points addressed in https://github.com/buildpack/spec/pull/42/commits/0f76577dc60ab6d06cfb9ca08600988288602d11